sdks/python/apache_beam/runners/portability/flink_runner_test.py

```diff
@@ -53,361 +54,386 @@
 from apache_beam.transforms import userstate
 from apache_beam.transforms.sql import SqlTransform
 
+# Run as
+#
+# pytest flink_runner_test.py \
+#     [--test_pipeline_options "--flink_job_server_jar=/path/to/job_server.jar \
```

Comment:
--test-pipeline-options

Reply:
Oh yeah, `--test_pipeline_options` is now required (even though it should be possible to leave it empty). Boyuan, would you mind filing a PR to fix this?